firebase-ml-modeldownloader/firebase-ml-modeldownloader.gradle

```diff
@@ -14,13 +14,29 @@
 
 plugins {
     id 'firebase-library'
+    id 'com.google.protobuf'
 }
 
 firebaseLibrary {
     testLab.enabled = false
     publishJavadoc = false
 }
 
+protobuf {
+    // Configure the protoc executable
+    protoc {
+        // Download from repositories
+        artifact = "com.google.protobuf:protoc:$protocVersion"
+    }
+    generateProtoTasks {
+        all().each { task ->
+            task.builtins {
+                java { }
```

Comment:
If I add that then - JsonFormat.parser().merge fails (based on testLogRequest_jsontToProto) which doesn't use lite option. Can you suggest an alternative?

Reply:
Oh, right. JsonFormat parser actually requires the full runtime to work. Since it's used for tests, I think you can do the same crashlytics is doing https://github.com/firebase/firebase-android-sdk/blob/master/firebase-crashlytics/firebase-crashlytics.gradle#L88

just use `testImplementation` instead of `androidTestImplementation`

Reply:
removed all the androidTestImplementation - these were not used. The encoding was already there for the regular implementation level (and validated it is needed at that level)

Reply:
AFAIK you can just get rid of the whole generateProtoTasks sections.

Reply:
Appears required - gives error when removed:
 Execution failed for task ':firebase-ml-modeldownloader:generateDebugUnitTestProto'.
 protoc: stdout: . stderr: Missing output directives.

Reply:
Got it. Thanks for following through!